js/src/util/wallet.js

```diff
@@ -75,7 +76,7 @@ export class Wallet {
   signTransaction (transaction) {
     const tx = new Transaction(transaction);
     tx.sign(this.seed);
```

Comment:
We *could* use `ethereumjs-util` directly (slightly more code in our codebase, but smaller dependency tree), as [`Transaction.prototype.sign` just calls two functions](https://github.com/ethereumjs/ethereumjs-tx/blob/0045465/index.js#L181).